Yoast/wordpress-seo
# Apply Select2 to all select boxes on breadcrumbs page

## Summary

This PR can be summarized in the following changelog entry:

* Apply Select2 to _all_ select boxes on breadcrumbs page.

## Test instructions

This PR can be tested by following these steps:

* Make sure you have several post types that support breadcrumbs.
* Go to Search Appearance → Breadcrumbs, see multiple dropdowns with breadcrumbs.

## Quality assurance


Fixes #8304
